pkg/ebpf/c/common/capabilities.h

```diff
@@ -0,0 +1,36 @@
+#ifndef __COMMON_CAPABILITIES_H__
+#define __COMMON_CAPABILITIES_H__
+
+#include <vmlinux.h>
+#include <vmlinux_flavors.h>
+
+#include <common/common.h>
+
+// BEFORE: Currently, (2021), there are ~40 capabilities in the Linux kernel
+// which are stored in an u32 array of length 2. This might change in the (not
+// so near) future as more capabilities will be added. For now, we use u64 to
+// store this array in one piece
+//
+// NEW NOTE: Recently, (2023), kernel has started using an u64 for all
+// capabilities, instead of using variable u32 array. Use type flavors to
+// deal with that.
+//
+
+static __always_inline u64 credcap_to_slimcap(void *from)
+{
+    kernel_cap_t___older to = {0};
```

Comment:
Perhaps moving this to `if (bpf_core_field_exists(to.cap)) {` scope?

Reply:
I also thought about it, but then, having the type flavors here keeps there cleaner (which is the "business logic" for the event)... that is why I kept it here. No biggie, both would be correct.